### DISCUSSION TITLE
Sponsorship?

### DISCUSSION THREAD

**Discussion by RickCogley** [upptime/upptime#163] [General] [answered]
Hi - have you considered opening this app's GH sponsorship page?

**Comment by mdc54-dot** [comment]
No sir

**Comment by AnandChowdhary** [comment, marked as answer]
Thanks for the idea, @RickCogley! Currently, @upptime is entirely funded by @koj-co (https://koj.co). At this point, we're not looking for sponsorships, but you can help us with your ideas! 😄

**Reply by RickCogley** [reply]
Ok, will do!
